code/modules/atmospherics/machinery/pipes/simple.dm

```diff
@@ -17,6 +17,13 @@
 	construction_type = /obj/item/pipe/binary/bendable
 	pipe_state = "simple"
 
+	max_pressure = 35000
+	var/mutable_appearance/reinforced
+
+/obj/machinery/atmospherics/pipe/simple/New()
+	. = ..()
+	reinforced = mutable_appearance(icon, "reinforced")
```

Comment:
Why is this on simple pipes? Why have more then one of these? Why should this exist at all shouldn't it be an iconstate?

Reply:
ops i should move them to the reinforced path